src/libecalc/presentation/yaml/yaml_models/pyyaml_yaml_model.py

```diff
@@ -269,8 +273,15 @@ def facility_inputs(self):
         return self._internal_datamodel.get(EcalcYamlKeywords.facility_inputs, [])
 
     @property
-    def time_series(self):
-        return self._internal_datamodel.get(EcalcYamlKeywords.time_series, [])
+    def time_series(self) -> List[YamlTimeSeriesCollection]:
```

Comment:
Should do this for all values in yaml model IMO. Doing it here when the properties are used makes it possible to work with the yaml file without everything being valid

Reply:
so, to only actually parse accordign to schema when we use a property?

Reply:
yes, we can have a separate validate function if we need to make sure the full model is valid.

Reply:
We could also require the model to be valid. 

It would be nice if this class took care of it though. This class would then read the model, then we can get the properties in the model as yaml classes, removing the need to validate a lot of stuff in other places.